cocos/platform/CCFileUtils.cpp

```diff
@@ -1189,6 +1189,11 @@ bool FileUtils::createDirectory(const std::string& path)
 
 bool FileUtils::removeDirectory(const std::string& path)
 {
+    // FIXME: Why using subclassing? an interface probably will be better
+    // to support different OS
+    // FileUtils::removeDirectory is subclassed on iOS/tvOS
+    // and system() is not available on tvOS
+#if !defined(CC_PLATFORM_IOS)
```

Comment:
@ricardoquesada why add this line since iOS and Mac implement `FileUtils::removeDirectory` in `CCFileUtils-apple.cpp`. 

And the logic is error, because `CC_PLATFORM_IOS` is defined [here](https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/platform/CCPlatformConfig.h#L42).